website/data/docs-nav-data.json

```diff
@@ -111,17 +120,13 @@
         "path": "install-boundary/configure-workers"
       },
       {
-        "title": "Non-dev environments",
-        "path": "install-boundary/no-gen-resources"
+        "title": "Initialize Boundary",
+        "path": "install-boundary/initialize"
```

Comment:
I thought this might have been a clearer? Curious to hear thoughts

Reply:
I like this!